src/agents/policy_retriever_agent.py

```diff
@@ -0,0 +1,135 @@
+import os
+import sys
+sys.path.append("..")
+
+from langchain_core.prompts import ChatPromptTemplate
+from langchain_community.vectorstores import Chroma
+from openai import OpenAI, AzureOpenAI
+
+import config as cfg
+from helpers.embedding_helpers import OpenAIEmbeddingFunction
+
+# Environment setup
+os.environ['KMP_DUPLICATE_LIB_OK'] = 'True'
+
+# Fuse to blob storage to access chroma db and policy documents
+os.system('sudo blobfuse /home/azureuser/cloudfiles/code/blobfuse/rndaistoragemeldingen --tmp-path=/mnt/resource/blobfusetmp_meldingen --config-file=/home/azureuser/cloudfiles/code/blobfuse/fuse_connection_meldingen.cfg -o attr_timeout=3600 -o entry_timeout=3600 -o negative_timeout=3600 -o allow_other -o nonempty')
```

Comment:
This should not be in the code, this should be a separate script that you execute yourself ones before (or potentially _at_) starting up the server. It's just a fix around our own storage situation, not really related to this retriever or anything like that... Btw it's also something I do for my tool (I just always do it before starting up the server).
Either way, even if you really really want it here, all paths should not be hardcoded here, but brought from the config file, so that they can be easily adjusted centrally in case we have e.g. a change in storage structure or suddenly want to use another dataset

Reply:
I'll remove this and add an instruction that the command should be run prior to initialization of the tools